tests/stubtest_third_party.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+"""Test typeshed's third party stubs using stubtest"""
+
+import argparse
+import subprocess
+import sys
+import tempfile
+import toml
+import venv
+from pathlib import Path
+
+
+EXCLUDE_LIST = ["Flask", "pyaudio", "backports", "pkg_resources", "six", "aiofiles", "pycurl"]
+
+
+class StubtestFailed(Exception):
+    pass
+
+
+def run_stubtest(dist: Path) -> None:
+    with open(dist / "METADATA.toml") as f:
+        metadata = dict(toml.loads(f.read()))
+
+    # Ignore stubs that don't support Python 2
+    if not bool(metadata.get("python3", True)):
+        return
+
+    with tempfile.TemporaryDirectory() as tmp:
+        venv_dir = Path(tmp)
+        venv.create(venv_dir, with_pip=True, clear=True)
```

Comment:
Is this a bottleneck? We could probably use the same venv for everything.

Reply:
I don't think it's much of a bottleneck. We definitely don't want to install everything in one pip command, because then pip's dependency resolver will likely lose its mind. So if we're consigned to multiple pip commands, the venv creation and deletion itself is probably pretty fast. Maybe something to look into if we dramatically increase the number of packages on typeshed we'll find this is too slow, but for now, with the sharding across two jobs, things should overall be fast enough.